### PR TITLE
Feat/add trade fee in base and quote

### PR DESCRIPTION
**Pull request by cardosofede**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
Add the value of the trade fee in quote asset. This is very useful for reporting purposes since if the fees are paid in another token, the rate oracle can be used to translate this difference.

**Tests performed by the developer**:
Run a bot and see that everything is working.
Test the new solution with the dashboard 

**Tips for QA testing**:

### CODE REVIEW

File: hummingbot/connector/markets_recorder.py [hunk 1]
```diff
@@ -260,11 +266,11 @@ def _did_fill_order(self,
                     order_id=order_id,
                     trade_type=evt.trade_type.name,
                     order_type=evt.order_type.name,
-                    price=Decimal(
```
Comment (fengtality) [inline hummingbot/connector/markets_recorder.py:263]: Is this change needed?

**Review by fengtality — APPROVED** [review]
LGTM

Tested alongside https://github.com/hummingbot/dashboard/pull/15

**Review by nikspz — APPROVED** [review]
Tests performed:

- Cloned and install feature branch and PR6288
- Manually Created docker image successfully
- conda env create -f environment_conda.yml
- conda activate streamlit-apps
- streamlit run main.py
- Copied URL to browser
- Created symlink with ln -s /path/to/hummingbot/data data
- Review Strategy Performance tab now shows fees in USDT for binance spot
- Confirmed fees amount is showing correct amount

Note: for `binance perpetual` there's ongoing issue on dashboard's `Strategy Performance tab` with `AttributeError: module 'ccxt' has no attribute 'binance_perpetual'
`
